src/metrics_test.go

```diff
@@ -234,26 +237,35 @@ func TestPopulateMetrics(t *testing.T) {
 		t.Error(err)
 	}
 
-	integration := sdk.Integration{}
-	ms := integration.NewMetricSet("metricsTestSample")
+	integration, err := integration.New(integrationName, integrationVersion, integration.Args(&args))
```

Comment:
we don't need integration for the test. We can create a new metric set just from `metric` pkg with `metric.NewSet()`

For example: `ms := metric.NewSet("metricsTestSample", persist.NewInMemoryStore(), attr)`